test/integration/rendering-error-plan-failed/00-install.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: kudo.k8s.io/v1alpha1
+kind: Operator
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: invalid
+---
+apiVersion: kudo.k8s.io/v1alpha1
+kind: OperatorVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: invalid-v1
+  namespace: default
```

Comment:
Optional: drop the namespace here so it installs cleanly into the same namespace.